lib/command_line_mock.ex

```diff
@@ -0,0 +1,12 @@
+defmodule CommandLineMock do
```

Comment:
Is it a Elixir convention to have mocks adjacent to the module counterparts? or should it be in the test directory?

Reply:
Just paired with Eli quickly and we figured out a way to have the mock remain in test using the `Code.require_file` in the test_helper.exs file. Now Im working on having it compile a mock directory instead of specifically this mock file so that future mocks can be easily added! Thank you for all the awesome advice and help! 🥳

Reply:
Just figured out how to traverse through a mock directory, grabbing each file and using the `Code.require_file` on each. Updating it now! :D

Reply:
Thanks for clarifying Jon!